### PR TITLE
Add storage-secret-store, resizer image, and node label image

### PR DESCRIPTION
**Pull request by dobsonj**
This pulls in the latest changes from https://github.com/IBM/ibm-vpc-block-csi-driver-operator master that are needed for the driver and operator to ship in 4.10. In particular, these two changes are essential:
https://github.com/IBM/ibm-vpc-block-csi-driver-operator/pull/8
https://github.com/IBM/ibm-vpc-block-csi-driver-operator/pull/10
Additionally, this PR includes:
- bumping .ci-operator.yaml to golang-1.17
- go mod tidy && go mod vendor
- make update-gofmt

/cc @openshift/storage

### CODE REVIEW

File: vendor/bou.ke/monkey/LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Copyright Bouke van der Bijl
+
+I do not give anyone permissions to use this tool for any purpose. Don't use it.
+
+I’m not interested in changing this license. Please don’t ask.
```
Comment (bertinatto) [inline vendor/bou.ke/monkey/LICENSE.md:5]: hm, this is a show-stopper

**Comment by dobsonj** [comment]
/hold
to sort out licensing issue that Fabio pointed out above.

File: vendor/bou.ke/monkey/LICENSE.md (hunk 1 above)
Comment (dobsonj) [reply vendor/bou.ke/monkey/LICENSE.md]: Good catch, this library was only being used in the unit test to patch out a method. I just pushed a small refactor to remove this dependency.

**Review by bertinatto — COMMENTED** [review]
/lgtm

**Pull request merged**